README.md

```diff
@@ -1,53 +1,58 @@
 
-When using HACS there is no need to manually make Home Assistant aware of custom javascript resources of this custom card. If you do need to check this manually, it is visible in the **Configuration** -> **Lovelace Dashboards** -> **Resources** tab.
+2. Follow the HACS [installation instructions](https://hacs.xyz/docs/installation/prerequisites). Then, open HACS, go to Frontend and click "Explore & Download Repositories" and search for "Valetudo Map Card". Select it and choose "Download".
 
-On older versions you may need to add this to the lovelace configuration yaml manually.  
-`lovelace.yaml`: Add custom Lovelace configuration in Home Assistant to enable valetudo-map-card JavaScript. Go to your HA overview, then right top "Configure UI", then right top again on the 3 dots and "Raw config editor".
-```yaml
-resources:
-  - type: js
-    url: /community_plugin/lovelace-valetudo-map-card/valetudo-map-card.js
-```
-### Home Assistant
+## Configuration
 
-`configuration.yaml`: Valetudo officially supports MQTT, with the preferred example configuration as follows. You will need to have MQTT configured in [Home Assistant](https://www.home-assistant.io/docs/mqtt/broker) and [Valetudo](https://hypfer.github.io/Valetudo/pages/integrations/home-assistant-integration.html).
+### Dashboard resources
 
-Then, in Home Assistant, go to `Configuration` -> `Devices and Services` -> `MQTT` and choose the entities. Then filter the entities on `vacuum.`, you will see a result like the following:
+When using HACS, there is no need to manually make Home Assistant aware of custom JavaScript resources of this custom card. Navigate to [**Settings** -> **Lovelace Dashboards** -> **Resources**](https://my.home-assistant.io/redirect/lovelace_resources/) in the web UI to verify the registration was succesfull. If no entry ending in `/valetudo-map-card.js` is present, it failed for some reason.
 
-![An example showing a single device named vacuum.valetudo_openidenticalwasp](https://user-images.githubusercontent.com/1885159/148192473-7e171869-5b52-4925-a326-25025c6f549b.png)
+To manually register the resource, either click **Add Resource** via the web UI and add the URL `/local/community/lovelace-valetudo-map-card/valetudo-map-card.js`, or add the following `data.item` to the `.storage/lovelace_resources` JSON file:
 
-In this case, the vacuum name is `valetudo_openidenticalwasp`. Obviously, the name for your vacuum will be different, but it should also be lowercase and starting with `valetudo_`.
+```json
+{
+  "url": "/local/community/lovelace-valetudo-map-card/valetudo-map-card.js",
+  "type": "module",
+  "id": "27308708ba994e5396c017d9e7d0a54c"
+}
+```
```

Comment:
I don't understand this, as my file has completely different values:

```
      {
        "url": "/hacsfiles/lovelace-valetudo-map-card/valetudo-map-card.js?hacstag=2030361082022012",
        "type": "module",
        "id": "b0109d6321cc47d59465ca8475efa84e"
      },
```

Reply:
Oops... actually, I don't understand either how the `id` and the `?hacstag=` param are built. I just copy-pasted what HA inserted when using the GUI to manually add the resource. And I was tired when I did, not really realizing what exactly I was copying... 🙈

I removed the part about manually editing `.storage/lovelace_resources`.